tests/gdk/commands/test/config/test_InitConfiguration.py

```diff
@@ -152,7 +152,7 @@ def test_GIVEN_gdk_config_with_otf_version_WHEN_test_init_with_otf_version_arg_T
         )
         self.mocker.patch("gdk.common.configuration.get_configuration", return_value=config)
         init_config = InitConfiguration({"otf_version": "1.2.3"})
```

Comment:
this file is still full of references to otf_version and otf in the test naming scheme, which I assume you are planning to fix?

Reply:
The ones that were left were intentional, since they're referring to properties in the config file or arguments for the commands. We do still support "otf" values in these scenarios (to keep support for anyone still using the old values), and some of the tests actually directly test the scenario where both otf and gtf keys are provided and we verify that the gtf ones take precedence.